Library/Homebrew/cask/cask.rb

```diff
@@ -127,6 +127,23 @@ def config_path
       metadata_main_container_path/"config.json"
     end
 
+    def download_sha_path
+      metadata_main_container_path/"LATEST_DOWNLOAD_SHA256"
+    end
+
+    def new_download_sha
+      require "cask/installer"
+
+      @new_download_sha ||= Installer.new(
+        self, verify_download_integrity: false
+      ).download(quiet: true).instance_eval { |x| Digest::SHA256.file(x).hexdigest }
```

Comment:
```suggestion
      @new_download_sha ||= Installer.new(
        self, verify_download_integrity: false
      ).download(quiet: true)
       .instance_eval { |x| Digest::SHA256.file(x).hexdigest }
```

Reply:
I think I'll just line up `.new`, `.download` and `.instance_eval` vertically instead.